tests/lang_callable/generics/tlate_bound_typedesc.nim

```diff
@@ -0,0 +1,15 @@
+discard """
+  errmsg: "The default parameter 'arg' has incompatible type with the explicitly requested proc instantiation"
+  line: 12
+"""
+
+# original issue https://github.com/nim-lang/nim/issues/11660
+
+# XXX: this semantics of ``typedesc`` as the parameter type are very likely
```

Comment:
This is a good description of the issue, and when the behaviour is fixed the test will be a good reminder.